### ISSUE TITLE
Search in SPS

### ISSUE THREAD

**Issue by hossamfawzy96** [SEGUC17/Foobar#27]
1: Severity: ( medium)

2:Reported: by Hossam

3:Description: on trying search without entering any values the website doesnot respond with no results or empty search value

4: Steps to reproduce the issue: SPS from home menu, then search with empty value

5: Expected result: displaying any error "no results" or "no search value"

**Comment by markarsanious** [comment]
yeah because it responds with the full list of  SPs. That's intended, if you tried to search on facebook with empty search bar it won't give a no results definitely.
